CHANGELOG.md

```diff
@@ -6,111 +6,86 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 # Release Notes
 
-## [Preview Release 6.0.0-preview3.24332.3] - 2024-11-27
```

Comment:
I think we should keep the preview versions documented in here.

Reply:
The changelog doesn't need to duplicate the list of changes, if nothing's changed, it can simply say so. Only 6.0.0.md needs to provide summarized view of all changes, which will be published on GitHub release.